File: filedata/entity/c#/Baserole.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 系统角色表
    /// </summary>
    [Table("Base_Role")]
    public class Baserole
    {
        
        /// <summary>
        /// 主键
        /// </summary>
        [Column("Id")]
        public string Id { get; set; }
    
    
        /// <summary>
        /// 创建时间
        /// </summary>
        [Column("CreateTime")]
        public DateTime CreateTime { get; set; }
    
    
        /// <summary>
        /// 创建人Id
        /// </summary>
        [Column("CreatorId")]
        public string CreatorId { get; set; }
    
    
        /// <summary>
        /// 否已删除
        /// </summary>
        [Column("Deleted")]
        public int Deleted { get; set; }
    
    
        /// <summary>
        /// 角色名
        /// </summary>
        [Column("RoleName")]
        public string RoleName { get; set; }
    
    
    }
}

File: filedata/entity/java/BaseRole.java

package entity;
// 系统角色表
public class Baserole  extends BaseDomain{
    public Baserole() {

    }
    
    //系统角色表
    private String Id;
        
    //系统角色表
    private Date CreateTime;
        
    //系统角色表
    private String CreatorId;
        
    //系统角色表
    private tinyint(4) Deleted;
        
    //系统角色表
    private String RoleName;
        
    public String getId() {
        return Id;
    }
    public void setId(String Id) {
        this.Id = Id;
    }
            
    public Date getCreatetime() {
        return Createtime;
    }
    public void setCreatetime(Date Createtime) {
        this.Createtime = Createtime;
    }
            
    public String getCreatorid() {
        return Creatorid;
    }
    public void setCreatorid(String Creatorid) {
        this.Creatorid = Creatorid;
    }
            
    public tinyint(4) getDeleted() {
        return Deleted;
    }
    public void setDeleted(tinyint(4) Deleted) {
        this.Deleted = Deleted;
    }
            
    public String getRolename() {
        return Rolename;
    }
    public void setRolename(String Rolename) {
        this.Rolename = Rolename;
    }
             
    @Override
    public String toString() {
        return "Base_Role[Id=" + Id + ",CreateTime=" + CreateTime + ",CreatorId=" + CreatorId + ",Deleted=" + Deleted + ",RoleName=" + RoleName + "]";
    }
        
    
}
        